packages/test/test-end-to-end-tests/src/test/gc/gcStats.spec.ts

```diff
@@ -29,7 +30,6 @@ const ensureContainerConnectedWriteMode = async (container: Container) => {
 		await new Promise<void>((resolve) =>
 			container.on("connected", () => resolveIfActive(resolve)),
 		);
-		container.off("connected", resolveIfActive);
```

Comment:
Why does this need to be removed?

Reply:
the code is wrong. in the on a new method is created, so the off does nothing. the off also has the wrong method signature in the listener. this worked on container which supports (string, (any)=>void), but IContainer does not